lib/bloc/reverse_swap_in/reverse_swaps_in_bloc.dart

```diff
@@ -0,0 +1,45 @@
+import 'dart:async';
+
+import 'package:breez_sdk/breez_bridge.dart';
+import 'package:breez_translations/breez_translations_locales.dart';
+import 'package:c_breez/bloc/reverse_swap_in/reverse_swaps_in_state.dart';
+import 'package:c_breez/utils/exceptions.dart';
+import 'package:fimber/fimber.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class ReverseSwapsInBloc extends Cubit<ReverseSwapsInState> {
+  final _log = FimberLog("ReverseSwapsInBloc");
+  final BreezSDK _breezLib;
+  late Timer timer;
+
+  ReverseSwapsInBloc(this._breezLib) : super(ReverseSwapsInState(isLoading: true)) {
+    pollReverseSwapsInProgress();
```

Comment:
We listen to notifications on Breezmobile to trigger fetches and since we don't have these notifications on SDK I assumed they would need to be fetch periodically only when the user is on ReverseSwapPage, similar to how SwapInProgress works on Receive via BTC Address page.

Reply:
When we're on Send to BTC Address page and have reverse swaps in progress, we display ReverseSwapsInProgress page instead and show their TX information. Polling allows us to automatically switch back to Send to BTC Address page after those tx's complete.